packages/react/src/components/Breadcrumbs/BreadcrumbsList.tsx

```diff
@@ -0,0 +1,33 @@
+import { inner, useMergeRefs } from '@floating-ui/react';
+import { Slot } from '@radix-ui/react-slot';
+import cl from 'clsx/lite';
+import { type HTMLAttributes, forwardRef, useEffect, useRef } from 'react';
+
+export type BreadcrumbsListProps = HTMLAttributes<HTMLOListElement>;
+
+export const BreadcrumbsList = forwardRef<
+  HTMLOListElement,
+  BreadcrumbsListProps
+>(({ className, ...rest }, ref) => {
+  const innerRef = useRef<HTMLOListElement>(null);
+  const mergedRefs = useMergeRefs([innerRef, ref]);
+
+  // Set aria-current on last link
+  useEffect(() => {
+    const links = innerRef.current?.querySelectorAll('a') || [];
+    const lastLink = links[links?.length - 1];
+    lastLink?.setAttribute('aria-current', 'page');
+
+    return () => lastLink?.removeAttribute('aria-current'); // Remove on re-render as React can re-use DOM elements
+  }, [rest.children]);
```

Comment:
Just a thought:
Would it be bad to drop the `useEffect` here?
I think we would be able to run this code on every render without any hiccups 🤔 

Since the children here should close to never re-render, we ensure that the `aria-current` is always correct, since updating props of the children might not trigger the effect to run again